src/lib/data/creatables/slayer.ts

```diff
@@ -105,6 +105,14 @@ export const slayerCreatables: Createable[] = [
 		outputItems: new Bank({ 'Uncharged toxic trident': 1 }),
 		GPCost: 0
 	},
+	{
+		name: 'Uncharged trident',
+		inputItems: new Bank({
+			'Trident of the seas (full)': 1,
+		}),
+		outputItems: new Bank({ 'Uncharged trident': 1 }),
```

Comment:
```suggestion
		outputItems: new Bank({
			'Uncharged trident': 1,
			'Death rune': 2500,
			'Chaos rune': 2500,
			'Fire rune': 12_500,
		}),
```
Missed a few output items
![image](https://user-images.githubusercontent.com/68188574/196494018-88ac4cdd-f9ad-4f4a-b64e-ce827e5e9c78.png)

Reply:
> @Lajnux should this be moved to the revertables then and treated like the blowpipe?

if you add a way to create Trident of the seas then yes you can just make the opposite called revert Trident of the seas